alpha/veneer/semver/semver.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/operator-framework/operator-registry/pkg/image"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/yaml"
+	"sigs.k8s.io/kind/pkg/errors"
```

Comment:
Can we use the "k8s.io/apimachinery/pkg/util/errors" package instead, and use that package's NewAggregate functionality? I think we'll want to move off having a sigs.k8s.io/kind direct dependency given it's only used for testing reasons, and minimizing the number of packages that use that dependency can help ease that process over time.

Reply:
I thought we always use apimachinery error. I don't think we use kind error anywhere. Probably just a wrong reference here.

Reply:
Can do.  I cribbed this approach from other code and wasn't sensitive to the import used.